node/cn/tracers/api.go

```diff
@@ -297,15 +329,17 @@ func (api *CommonAPI) traceChain(start, end *types.Block, config *TraceConfig, n
 		tasks    = make(chan *blockTraceTask, threads)
 		results  = make(chan *blockTraceTask, threads)
 		localctx = context.Background()
+		reler    = new(releaser)
```

Comment:
```suggestion
		releaser    = new(releaser)
```

Reply:
The name shadows the struct `releaser`. I think original name is quite okay.